openupgrade_scripts/scripts/hr_holidays/14.0.1.5/pre-migration.py

```diff
@@ -4,11 +4,13 @@
 from openupgradelib import openupgrade
 
 _column_copies = {
-    "hr_leave_type": [("validation_type", "allocation_validation_type", "varchar")]
+    "hr_leave_type": [("validation_type", "allocation_validation_type", "varchar")],
+    "hr_leave": [("name", "private_name", "varchar")],
 }
 
 _field_renames = [
     ("hr.leave.type", "hr_leave_type", "validation_type", "leave_validation_type"),
+    ("hr.leave", "hr_leave", "name", "private_name"),
```

Comment:
```suggestion
    ("hr.leave", "hr_leave", "name", "private_name"),
```
do the same with hr.leave.allocation

Reply:
Done, thank you